src/states/elder/tests.rs

```diff
@@ -413,7 +469,7 @@ fn new_elder_state(
     let parsec_map = ParsecMap::new(full_id.clone(), gen_pfx_info);
     let chain = Chain::new(min_section_size, public_id, gen_pfx_info.clone());
     let peer_map = PeerMap::new();
-    let peer_mgr = PeerManager::new(public_id, true);
+    let peer_mgr = PeerManager::new(public_id, false);
```

Comment:
why the flag needs to be changed from `true` to `false`?

Reply:
This flag control whether client IP limiting is enabled (`true` = disable, `false` = not disable). I needed it to be enabled for the new tests to work. It doesn't affect the tests that were already there.